integration_tests/mockApis/manageUsersApi.ts

```diff
@@ -49,7 +47,7 @@ const ping = () =>
   })
 
 export default {
-  stubManageUser: (name = 'john smith'): Promise<[Response, Response]> =>
-    Promise.all([stubUser(name), stubUserRoles()]),
```

Comment:
the manage user implementation doesn't call user roles, so there is no point stubbing it.  Instead we expose the stub below